manifest.json

```diff
@@ -24,13 +24,13 @@
 "content_scripts": [
     {
         "matches": [
-        "*://*.app.slack.com/*"
+        "*://*.docs.google.com/document/*"
```

Comment:
I'm including `/document` here so that the extension doesn't run on other Google docs pages like the home page.